couchbase/transactions/attempt_context.hxx

```diff
@@ -50,6 +50,16 @@ public:
    * @param id The unique id of the document.
    * @return The result of the operation, which is an @ref error and a @ref
    * transaction_get_result.
+   *
+   * The example below shows how to use custom transcoder with transactions.
+   *
+   * The type should have regular transcoder to be defined. The only important
```

Comment:
> The type should have regular transcoder to be defined.

It's not very clear to me what this means. Could we phrase it differently?

Reply:
I will try to rephrase it. What I meant is that the transactions are using the same transcoders as general API, just with only exception that they should either encode types as JSON or Binary with returning corresponding flags, encoding with different set of flags is not defined.

Reply:
Updated.